components/CommunityModerators/ModeratorsCard/ModeratorCard.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from "./ModeratorCard.module.css";
+
+const ModeratorsCard = ({ image, name, title }: any) => {
```

Comment:
Don't we have a type for image, name, and title?

Reply:
i fixed it